package.json

```diff
@@ -93,6 +100,8 @@
     "@types/lodash.get": "^4.4.6",
     "@types/lodash.isequal": "^4.5.5",
     "@types/react-router-dom": "^5.1.3",
+    "eslint-plugin-jest": "^23.11.0",
+    "eslint-plugin-react-hooks": "^4.0.2",
```

Comment:
eslint-plugin-react-hooks is already enabled by create-react-app, no? I totally remember getting those.

Reply:
Good spot!